WalletWasabi/Services/SatoshiSynchronizer.cs

```diff
@@ -129,7 +137,7 @@ async Task StartReceivingMessagesAsync(WebSocket ws)
 						var filter = reader.ReadFilterModel();
 						if (localChain.TipHeight + 1 != filter.Header.Height)
 						{
-							Logger.LogError(ChainHeightMismatchError(filter));
+							Logger.LogDebug(ChainHeightMismatchError(filter));
```

Comment:
```suggestion
							Logger.LogInfo(ChainHeightMismatchError(filter));
```

Can you use `Info`? It might help us find problem client side linked to reorgs.